dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -174,3 +174,6 @@ getRealPathReturnsQualifiedPath.desc=Return non-null paths from the ServletConte
 
 stopAppStartUponListenerException.name=Stop application for unhandled listener exception
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. Set this property to true to stop the application from starting up when an unhandled exception is thrown from the context listeners.
+
+redirectToRelativeUrl.name=Send response redirect to a relative URL location
+redirectToRelativeUrl.desc=Send response redirect to a relative URL location without processing it. Set this property to true to send response redirect without checking and converting to an absolute URL location. 
```

Comment:
sounds good

Reply:
based on above comments, should be `redirect response`. I'm also wondering if "checking" is needed here, or if it might be part of "converting". If not:

`Set this property to true to send a redirect response without converting the URL to an absolute location.`

Reply:
sounds good!